packages/11ty/_plugins/iiif/process/index.js

```diff
@@ -3,8 +3,11 @@ const path = require('path')
 const addGlobalData = require('./addGlobalData')
 const initCreateImage = require('./createImage')
 const initCreateManifest = require('./createManifest')
+const initLogging = require('../../../_lib/chalk')
```

Comment:
Does it the implementation more clear to find a name for this import that makes it implicit this is factory function?

Reply:
`chalkFactory` is explicit though were we expand the implementation it may become less so.